pom.xml

```diff
@@ -91,6 +91,12 @@
     <maven-scm.version>1.9.5</maven-scm.version>
     <!-- To skip the wizard by default in hpi:run. It should not impact other goals. -->
     <hudson.Main.development>true</hudson.Main.development>
+    <!--
+      Starting with 2.33, jenkins-test-harness puts a space character in the path of the temporary workdir of agents
+      created from JenkinsRule.  It can help catching shell quoting bugs.  If your plugin's tests break because of this
+      and it can't be fixed, then you can disable this behavior by setting "jenkins.test.noSpaceInTmpDirs" to "true".
```

Comment:
The comment rubs me the wrong way. How is this not fixable?

I mean, sure, you may want (or need) a new parent POM for some reason, and want to release changes without addressing the new hoops the POM makes you jump through, but ultimately the goal is to get you to fix your broken plugin.

Disabling all the quality checks is definitely an anti-pattern, and I don't think we should make it this easy.